src/extension/background-script/connectors/alby.ts

```diff
@@ -254,11 +254,22 @@ export default class Alby implements Connector {
         token: this.config.oAuthToken, // initialize with existing token
       });
 
+      //// Currently the JS SDK guarantees request of a new refresh token is done synchronously.
```

Comment:
I think this comment is in the wrong place

Can you put it in the `authClient.on("tokenRefreshFailed")` function?